### PR TITLE
Changed the order of multiplication when calculating quaternion based…

### PR DESCRIPTION
**Pull request by urmahp**
… vel and acc

This is done since the velocity and acceleration is given in the body-local reference frame.

The math supporting this change, can be found [here](https://arxiv.org/abs/0811.2889v1).

I will try to think about a test, that could cover these changes.

### CODE REVIEW

**Comment by urmahp** [comment]
Added test to cover the changes.  

I don't know why the format check is complaining...

File: cartesian_trajectory_interpolation/include/cartesian_trajectory_interpolation/cartesian_trajectory_segment.h [hunk 1]
```diff
@@ -142,7 +142,8 @@ std::ostream& operator<<(std::ostream& os, const CartesianTrajectorySegment::Spl
  *
  * The computation of quaternion velocities and accelerations from
  * Cartesian angular velocities and accelerations is based on
- * <a href="https://math.stackexchange.com/questions/1792826">this blog post</a>.
+ * <a href="https://math.stackexchange.com/questions/1792826">this blog post</a> and on <a
+ * href="https://arxiv.org/abs/0811.2889v1">this article</a>.
```
Comment (stefanscherzinger) [inline cartesian_trajectory_interpolation/include/cartesian_trajectory_interpolation/cartesian_trajectory_segment.h:146]: @urmahp Could be a trailing white space that causes the linting to fail..

Comment (urmahp) [reply cartesian_trajectory_interpolation/include/cartesian_trajectory_interpolation/cartesian_trajectory_segment.h]: Yes that was the issue, thank you.

**Comment by urrsk** [comment]
IT seems this have been forgotton!
Will someone review this?

**Review by fmauch — APPROVED** [review]
Thank you for taking care about this @urmahp

**Pull request merged**